content/en/docs/concepts/containers/images.md

```diff
@@ -362,7 +278,7 @@ Kubernetes supports specifying container image registry keys on a Pod.
 
 #### Creating a Secret with a Docker config
 
-Run the following command, substituting the appropriate uppercase values:
+Using a Docker registry as an example, run the following command, substituting the appropriate uppercase values:
```

Comment:
```suggestion
You need to know the username, registry password and client email address for authenticating
to the registry, as well as its hostname.
Run the following command, substituting the appropriate uppercase values:
```

The “Docker” bit here is that the contents of the Secret are compatible with Docker. The container registry doesn't have to be Docker Hub.